Feature/projectile particles

Jazy5552 opened the pull request:
Added on wall hit effects to projectiles. We can expand on this and add effects to when an enemy or player is hit.

Jazy5552 committed:
Create particle effect component for projectiles

Jazy5552 committed:
Apply hit effects to current projectiles

Jazy5552 committed:
Update postprocessing

Jazy5552 committed:
Update my sandbox

AleksandrHovhannisyan reviewed (dismissed):
Very nice!

Only two things:

- Does postprocessing currently do anything? Didn't notice a difference except that the level's darker.
- When you launch your scene, there's a momentary particle emission. Did you notice this?

Jazy5552 commented:
Postprocessing is not used in the game right now. I just accidentally updated so figured I'd just commit it.

> When you launch your scene, there's a momentary particle emission. Did you notice this?

Yeah I was testing the particle system, I forgot to remove it. Should be fine.

Jazy5552 committed:
Resolve merge conflict with my branch

Merge branch 'develop' of https://github.com/cap4053-cheeky-pixels/EmbodyGame into feature/projectile-particles

 Conflicts:
	Assets/Scenes/SandboxScenes/JazySandbox.unity
